node/src/transition_frontier/genesis/transition_frontier_genesis_config.rs

```diff
@@ -130,6 +135,44 @@ impl GenesisConfig {
                 };
                 (mask, load_result)
             }
+            Self::DaemonJson(config) => {
+                let genesis_timestamp = config
+                    .genesis
+                    .as_ref()
+                    .map(|g: &daemon_json::Genesis| {
+                        g.genesis_state_timestamp()
+                            .map(|t| t.0.0.0)
+                    })
+                    .transpose()?
+                    .unwrap_or_else(|| {
+                        SystemTime::now()
+                            .duration_since(UNIX_EPOCH)
+                            .unwrap()
+                            .as_millis() as u64
+                    });
+                let constants = Self::default_constants(genesis_timestamp);
+                let ledger = config
+                    .ledger
+                    .as_ref()
+                    .ok_or(anyhow::anyhow!("No ledger in config"))?;
+                let account_list = ledger
+                    .accounts
+                    .as_ref()
+                    .ok_or(anyhow::anyhow!("No accounts in config"))?;
+                let accounts: Vec<Account> = account_list
+                    .iter()
+                    .map(daemon_json::Account::to_account)
+                    .collect::<Result<Vec<Account>, AccountConfigError>>()
+                    .map_err(anyhow::Error::from)?;
+                let (mut mask, total_currency) = Self::build_ledger_from_accounts(accounts);
+                let result = GenesisConfigLoaded {
+                    constants,
+                    ledger_hash: ledger_hash(&mut mask),
```

Comment:
@Sventimir which hash are you getting here when using the daemon.json file? issue could be that you are not inserting the winner account (check the ledger tool and how it generates it)

https://github.com/openmina/openmina/blob/develop/tools/ledger-tool/src/main.rs#L128-L138

Reply:
I added the genesis winner account in https://github.com/openmina/openmina/pull/369/commits/629b87e8eaa99414634b19ab9311ba837b46b4cc, but it does not seem to help much. I still get the same error.

Reply:
@Sventimir try adding the genesis account first, then the rest

Reply:
Did that, didn't help either. The ledger hash I get is different than expected: `jxA9yKKm77xdUbCBp59bdaTVRD9MqzX8HorKyg5sghFQeckUxnL`.